Update best-practices.md

veena-vee989 opened the pull request:
@WrathfulSpatula I've updated this doc with Jordan's new examples. 

I don't have the right level of access to upload photos, so I was only able to include the language and indicate where the images need to be. Will send you the images on Discord. 

Thank you, Dan!!

crazy4pi314 reviewed (approved):
LGTM! Thanks for adding the project 💖

nathanshammah commented:
Thanks @vprusso, looking forward to it!

veena-vee989 committed:
Update best-practices.md

@WrathfulSpatula I've updated this doc with Jordan's new examples. 

I don't have the right level of access to upload photos, so I was only able to include the language and indicate where the images need to be. Will send you the images on Discord. 

Thank you, Dan!!

veena-vee989 committed:
Update best-practices.md
